lib/app_profiler/railtie.rb

```diff
@@ -28,11 +28,14 @@ class Railtie < Rails::Railtie
         "APP_PROFILER_SPEEDSCOPE_URL", "https://speedscope.app"
       )
       AppProfiler.profile_header = app.config.app_profiler.profile_header || "X-Profile"
+      AppProfiler.profile_async_header = app.config.app_profiler.profile_async_header || "X-Profile-Async"
       AppProfiler.profile_root = app.config.app_profiler.profile_root || Rails.root.join(
         "tmp", "app_profiler"
       )
       AppProfiler.context = app.config.app_profiler.context || Rails.env
       AppProfiler.profile_url_formatter = app.config.app_profiler.profile_url_formatter
+      AppProfiler.upload_queue_max_length = app.config.upload_queue_max_length || 10
```

Comment:
this and the next one are wrong, it should be `app.config.app_profiler....` missed `app_profiler`